procedures/katello/branch.md.erb

```diff
@@ -52,6 +52,10 @@
   - [ ] [pulpcore](https://github.com/theforeman/puppet-pulpcore)
   - [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
   - [ ] [katello](https://github.com/theforeman/puppet-katello)
+- [ ] Create PRs to [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) to add a mapping between the current Foreman and Katello release branches and update pipelines. See [this test mapping PR example](https://github.com/theforeman/jenkins-jobs/pull/179) and [this pipeline PR example](https://github.com/theforeman/jenkins-jobs/pull/178).
+  - [ ] Files to edit: [testKatello.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/test/testKatello.groovy) and [katello-pipelines.yml](https://github.com/theforeman/jenkins-jobs/blob/master/centos.org/jobs/katello-pipelines.yml)
+  - [ ] File to create: [<%= release %>.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/katello/<%= release %>.groovy).  Existing example: [4.3.groovy](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/katello/4.3.groovy)
```

Comment:
A common error is to base it on the existing version instead of nightly. This commonly means you can get incorrect OSes when one is added or removed. Please suggest to copy `nightly.groovy` and modify the version numbers in it instead.